cluster/gce/util.sh

```diff
@@ -1593,7 +1593,8 @@ function create-certs {
   local -r primary_cn="${1}"
 
   # Determine extra certificate names for master
-  local octets=($(echo "${SERVICE_CLUSTER_IP_RANGE}" | sed -e 's|/.*||' -e 's/\./ /g'))
+  local octets
+  kube::util::read-array octets < <(echo "${SERVICE_CLUSTER_IP_RANGE}" | sed -e 's|/.*||' -e 's/\./ /g')
```

Comment:
I'm not sure this is the same behavior? My read is that `read-array` assumes SERVICE_CLUSTER_IP_RANGE is multi-line, where the previous method allows for space-separated terms

Reply:
Thanks, as you probably noticed I am trying to smoke out what is the problematic change in the equation.
It sure looks like you are right. I will investigate.

Reply:
Bullseye, @spiffxp! I did an experiment and left this change out and now all tests pass nicely.
I will look into an updated (=working) solution tomorrow; it is getting late in this neck of the woods.